websites/R/Ryou Flashcards/presence.ts

```diff
@@ -0,0 +1,121 @@
+const presence = new Presence({
+    clientId: "794689032411283476"
+    }),
+
+    startTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+
+    const presenceData: PresenceData = {
+        largeImageKey:
+            "ryou",
+        startTimestamp
+        },
```

Comment:
what you use that it did json object like this? XD

Reply:
I am glad semid runs over this